backend/profileTable.sql

```diff
@@ -0,0 +1,14 @@
+-- @block
+CREATE TABLE IF NOT EXISTS profileTable(
+    accountID INT NOT NULL,
+    userName VARCHAR(255) NOT NULL,
```

Comment:
Is this the user's real name (i.e. `Bob Hasalastname`) or their display name (i.e. `pussyslayer69`)? If it's a display name, we might want to consider adding a column for their real name either here or in the accounts table.

Reply:
name is a reserved keyword that I can't use so I just chose username. Is it ambiguous? Should I use actualName or something like that.

Reply:
I think you can leave this as-is. I think it would be nice to have a sample record as a comment in these files, or maybe in some other reference document we're tracking for this project.